Main.java

```diff
@@ -1,3 +1,6 @@
+import java.util.Scanner;
+
+
 /**
  *
  * @author 

```

Comment:
Don't forget to fill out the code header. IT should include a description of what the program is and also your name.